### PR TITLE
Fix reactive extension

### PR DESCRIPTION
**Pull request by SzymonMrozek**

### CODE REVIEW

File: JSONStorage.swift [hunk 1]
```diff
@@ -20,6 +20,11 @@ public enum JSONStorageType {
     }
 }
 
+public enum UseReadMemoryCache {
+    case no
```
Comment (piotrbernad) [inline JSONStorage.swift]: I dont like the idea of having enum here. I would separate this to simple useReadMemoryCache and saveTimeInterval variables.

**Review by piotrbernad — CHANGES_REQUESTED** [review]

File: JSONStorage.swift (hunk 1 above)
Comment (SzymonMrozek) [reply JSONStorage.swift]: But `saveTimeInterval` makes no sense when `useReadMemoryCache` set to false. But I agree this enum is also poor solution

Comment (piotrbernad) [reply JSONStorage.swift]: So lets keep previous solution and make a comment about that this property is not used when useReadMemoryCache is false.

Comment (SzymonMrozek) [reply JSONStorage.swift]: Ok Bro! 😸

**Pull request merged**